tools/release-ci/cmd/cloneref.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/release/tools/release-ci/pkg/pullrefs"
+)
+
+// clonerefCmd clones a PULL_REFS specification
+var clonerefCmd = &cobra.Command{
+	Use:   "cloneref REPOSITORY_URL [PULL_REFS]",
+	Short: "Clones a PULL_REFS specification from Prow into the specified directory",
+	Long: `Clones a PULL_REFS specification from Prow into the specified directory
+	
+PULL_REFS is expected to be in the form of:
+
+base_branch:commit_sha_of_base_branch,pull_request_no:commit_sha_of_pull_request_no,...
```

Comment:
Hmm, sorry -- what i meant to say is we still probably want to know what SHA we are working with when we say "latest" -- otherwise it's going to be super hard to determine what version of X some PR was built with. So before this script is called we'd want to resolve that?